docs/output_screening.md

```diff
@@ -11,12 +11,11 @@ The directories listed below will be created in the results directory after the
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
 - [Preprocessing](#preprocessing)
-  - [FastQC](#sequences) - Input sequence preparation (reference, protospacer, template)
   - [FastQC](#fastqc) - Read Quality Control
 - [Counting](#counting)
   - [MAGeCK count](#mageck-count) - Mapping reads to reference
-- [CNV correction](#counting)
-  - [CRISPRcleanR](#crisprcleanr) - Copy Number Variation correction and read normalization in case of knock-out screens.
+- [CNV correction](#cnv-correction))
+  - [CRISPRcleanR](##crisprcleanr-normalization) - Copy Number Variation correction and read normalization in case of knock-out screens.
```

Comment:
```suggestion
  - [CRISPRcleanR](#crisprcleanr-normalization) - Copy Number Variation correction and read normalization in case of knock-out screens.
```